recipes/aiozmq/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "aiozmq" %}
+{% set version = "0.7.1" %}
+{% set sha256 = "b41d589a715acc0d42b525ef7084bbcdc03c986ef5a90e212ee7f086125cf6c0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<33]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - pyzmq >=13.1
+  run:
+    - python
+    - pyzmq >=13.1
+    - msgpack-python >=0.4.0
+
+test:
+  imports:
+    - aiozmq
+    - aiozmq.cli
+    - aiozmq.rpc
+
+  commands:
+    - aiozmq-proxy --help  # [not win]
```

Comment:
You might want to add this as an entrypoint

Reply:
It's already specified as an entrypoint in the package's [setup.py](https://github.com/aio-libs-abandoned/aiozmq/blob/v0.7.1/setup.py#L72).

Reply:
Made this *hopefully* more clear [above]( https://github.com/conda-forge/staged-recipes/pull/5561/files#r180284721 ).